<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="rootWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>570</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Simulation Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset Simulation</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <spacer name="simulationSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Light</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="1">
            <widget class="QSpinBox" name="heightSpinBox">
             <property name="minimum">
              <number>50</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
             <property name="value">
              <number>400</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="widthSpinBox">
             <property name="minimum">
              <number>100</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
             </property>
             <property name="value">
              <number>3000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="widthLabel">
             <property name="text">
              <string>Width: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="heightLabel">
             <property name="text">
              <string>Height: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="applyNewBoundsButton">
             <property name="text">
              <string>Update Bounds</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="showHideLightButton">
             <property name="text">
              <string>Show/Hide</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="SimulationViewWidget" name="simulationViewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="playSpeedGroupWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="playSpeedSpacerLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pauseButton">
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Pause.svg</normaloff>:/Pause.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="speed1Button">
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Play.svg</normaloff>:/Play.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">playSpeedButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="speed2Button">
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/FastForward.svg</normaloff>:/FastForward.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">playSpeedButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="speed3Button">
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/FastFastForward.svg</normaloff>:/FastFastForward.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">playSpeedButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="speedMaxButton">
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/MaxSpeed.svg</normaloff>:/MaxSpeed.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">playSpeedButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="playSpeedSpacerRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="playSpeedSpacerBottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimulationViewWidget</class>
   <extends>QWidget</extends>
   <header>SimulationViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="playSpeedButtonGroup"/>
 </buttongroups>
</ui>
